packages/core/src/external/fhir/adt-encounters.ts

```diff
@@ -137,7 +163,7 @@ export async function getAdtSourcedEncounter({
     `getAdtSourcedEncounter - cx: ${cxId}, pt: ${patientId}, enc: ${encounterId}`
   );
 
-  const fileKey = createFileKeyAdtLatest({
+  const fileKey = createFileKeyAdtEncounterSourceOfTruth({
```

Comment:
"source of truth" seems less clear than "latest". In theory, all of them are source of truth, each represent the data at that particular point in time/interaction with the external server - thus, "source of truth" applies to all of them.

"latest", OTOH, means this is the most up-to-date representation, and with that I know that I need this one, not the other files that represent the point in time intregration.

Reply:
clarified on slack, updated!